command_line/mono_pkg.yaml

```diff
@@ -1,6 +1,5 @@
 sdk:
-  - dev
-  - stable
```

Comment:
are you sure you want to lose stable?

Reply:
Planning to add this back after the release